packages/fxa-admin-panel/src/components/EmailBlocks/index.test.tsx

```diff
@@ -5,110 +5,151 @@
 import React from 'react';
 import Chance from 'chance';
 import { render, fireEvent, act } from '@testing-library/react';
-import { MockedProvider, MockedResponse } from '@apollo/react-testing';
+import { MockedProvider, MockedResponse, wait } from '@apollo/react-testing';
 import '@testing-library/jest-dom/extend-expect';
+import { CLEAR_BOUNCES_BY_EMAIL } from './Account/index';
 import { GET_ACCOUNT_BY_EMAIL, EmailBlocks } from './index';
 
 const chance = new Chance();
+let testEmail: string;
+let deleteBouncesMutationCalled = false;
 
-const exampleEmail = chance.email();
-
-function exampleEmailBounce() {
+function exampleBounce(email: string) {
   return {
-    email: exampleEmail,
+    email: email,
     createdAt: chance.timestamp(),
     bounceType: 'Permanent',
     bounceSubType: 'General',
   };
 }
 
-const exampleAccount: MockedResponse = {
-  request: {
-    query: GET_ACCOUNT_BY_EMAIL,
-    variables: {
-      email: exampleEmail,
+function exampleAccountResponse(email: string): MockedResponse {
+  return {
+    request: {
+      query: GET_ACCOUNT_BY_EMAIL,
+      variables: {
+        email,
+      },
     },
-  },
-  result: {
-    data: {
-      accountByEmail: {
-        uid: 'a1b2c3',
-        email: exampleEmail,
-        createdAt: chance.timestamp(),
-        emailBounces: [exampleEmailBounce(), exampleEmailBounce()],
+    result: {
+      data: {
+        accountByEmail: {
+          uid: 'a1b2c3',
+          email,
+          createdAt: chance.timestamp(),
+          emailBounces: [exampleBounce(email), exampleBounce(email)],
+        },
       },
     },
-  },
-};
+  };
+}
+
+function exampleBounceMutationResponse(email: string): MockedResponse {
+  return {
+    request: {
+      query: CLEAR_BOUNCES_BY_EMAIL,
+      variables: {
+        email: email,
+      },
+    },
+    result: () => {
+      deleteBouncesMutationCalled = true;
+      return {
+        data: {
+          clearEmailBounce: true,
+        },
+      };
+    },
+  };
+}
+
+beforeAll(function() {
```

Comment:
This doesn't matter of course but for consistency we could use an arrow function here and line 70🤷‍♀️